### ISSUE TITLE
handleOpenURL() not triggered with Cordova 5.0+

### ISSUE THREAD

**Issue by rshchang88** [EddyVerbruggen/Custom-URL-scheme#232]
The handleOpenURL worked fine for Cordova 4 and below.  It does not trigger with any Cordova version above 5.  When I moved it back to 4 it instantly worked.  One of my functions required 5+ Cordova.

This issue happens only with iOS.  Android works fine.  The app is launched when using the custom URL scheme for both platform, but in iOS it doesn't trigger handleOpenURL and executes the functions within, so it just sits there...

**Comment by FirewizzNL** [comment]
I Have the same issue, it just does't work on Cordova 5+
Really like to see a solution tho..

**Comment by pacedavis** [comment]
Same issue here, on Android.  Has anyone figured out a solution?

**Comment by adamalton** [comment]
Same issue on Cordova 7.0.1 for iOS.

**Comment by premtemp** [comment]
Same issue with cordova 6.5... Did anyone find a solution,,

**Comment by samiroquai** [comment]
I can confirm that I successfully tested the handleOpenURL with Cordova 7.1.0, on Android (cordova-android 6.2.3)

**Comment by AmyJUn** [comment]
yes ios not working mine is  cordova 6, is it still not working?
